RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.7.3
+
+- fixed: merge-base works for branches that have no common commit
```

Comment:
We still remain in 3.7.2! take a look at https://github.com/VirtusLab/git-machete/releases, 3.7.2 hasn't been released yet